src/test/regress/expected/multi_schema_support.out

```diff
@@ -448,33 +429,67 @@ CREATE COLLATION test_schema_support.english FROM "en_US";
 \c - - - :worker_2_port
 CREATE COLLATION test_schema_support.english FROM "en_US";
```

Comment:
Could you also add a select test similar to;

select title from articles_hash order by title COLLATE test_schema_support.english;